client/src/entrypoints/admin/userbar.js

```diff
@@ -403,13 +398,14 @@ class Userbar extends HTMLElement {
             const currentA11ySelectorText = currentA11ySelector.querySelector(
               '[data-a11y-result-selector-text]',
             );
+            const selectorName = node.target.toString();
```

Comment:
I think we should change this further, so we understand why this is an array, and process it accordingly. Either do a `node.target.join(', ')`, or something more complex if needed.